src/app-layout/__tests__/drawers.test.tsx

```diff
@@ -35,82 +34,14 @@ describeEachAppLayout(size => {
     const { wrapper } = renderComponent(<AppLayout contentType="form" {...emptyDrawerItems} />);
 
     expect(wrapper.findDrawersTriggers()).toHaveLength(0);
-    expect(wrapper.findToolsToggle()).toBeTruthy();
+    expect(wrapper.findToolsToggle()).toBeFalsy();
   });
 
-  test('should apply drawers treatment to the tools if at least one other drawer is provided', () => {
```

Comment:
This and other test from this file moved into `runtime-drawers.test.tsx`, because the tools+drawers merging feature is now only available for runtime drawers